src/test/java/pl/gov/coi/pomocua/ads/translations/TranslationResourceTest.java

```diff
@@ -0,0 +1,96 @@
+package pl.gov.coi.pomocua.ads.translations;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import pl.gov.coi.pomocua.ads.PageableResponse;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+class TranslationResourceTest {
```

Comment:
This begs for a base test class for both `jobs` and `translations`

Reply:
Done